### DISCUSSION TITLE
Chianet never sync after reboot

### DISCUSSION THREAD

**Discussion by mathgonzlez** [guydavis/machinaris#648] [Q&A]
**Describe the bug**
Never endind to sync after container reboot

**To Reproduce**

Steps to reproduce the behavior, e.g.:
1. Set up config with 'docker-compose in linux (rhel8)'
2. Run Machinaris with 'docker'

**Expected behavior**
chia space pool synced and farm

**System setup:**
 - OS: rhel 8.55
 - Docker version number : Docker version 20.10.12, build e91ed57
 - Machinaris branch: main
 - Machinaris version number: Machinaris 0.7.0

**Config**
version: '3.7'
services:
    machinaris:
        image: ghcr.io/guydavis/machinaris:latest
        container_name: machinaris
        hostname: chia-gigabyte
        restart: always
        volumes:
            - ~/.machinaris:/root/.chia
            - /chia/plots/1b/plots:/plots1
            - /chia/plots/2b/plots:/plots2
            - /chia/plots/3b/plots:/plots3
            - /chia/plots/5b/plots:/plots5
            - /chia/plots/6b/plots:/plots6
            - /chia/plots/7b/plots:/plots7
            - /chia/plots/8b/plots:/plots8
        environment:
            - TZ=America/Buenos_Aires
            - mode=fullnode
            - worker_address=192.168.50.30
            - plots_dir=/plots1:/plots2:/plots3:/plots5:/plots6:/plots7:/plots8
            - blockchains=chia
        ports:
            - 8926:8926
            - 8927:8927
            - 8444:8444
            - 8447:8447

**Additional context & screenshots**

Chia Netspace sometimes is bigger and sometimes comeback to less space size, the issues started when the pc rebooted and the container was restarted annd then never come online again. Check the name of screenshot, at 10.06 have less size that at 10.04

Screen Shot 2022-04-18 at 10.04.20
![image](https://user-images.githubusercontent.com/57609433/163812358-53cdf4da-90fd-4d46-a88a-8a0bd5e3626b.png)

Screen Shot 2022-04-18 at 10.06.21
![image](https://user-images.githubusercontent.com/57609433/163812577-08249445-437a-4f20-8ec3-8b078377a7ec.png)

Thanks!

**Comment by guydavis** [comment]
Hi, thanks for the detailed report.  Please try the following process to [restart the blockchain sync](https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-blockchain-database).  Hope this helps!
